docs/demo_features

```diff
@@ -0,0 +1,28 @@
+# Features
+
+- **[Kubernetes](https://kubernetes.io)**: the app is designed to run on
+  Kubernetes (both locally , as well as on the cloud) using a Helm chart.
+- **[Docker](https://docs.docker.com)**: this forked sample can also be executed
```

Comment:
```suggestion
- **[Docker](https://docs.docker.com)**: this forked demo can also be executed
```

Reply:
"this forked sample" -> the app
(to be aligned with the Kubernetes bullet)